dali/python/nvidia/dali/ops/_operators/python_function.py

```diff
@@ -99,6 +99,7 @@ def _dlpack_from_array(array):
 
 class PythonFunction(PythonFunctionBase):
     schema_name = "PythonFunction"
+    _generated = False
```

Comment:
I presume this is here to override the default `True` from the base class?

Reply:
It's here for completeness with the `_generated = True` from the generated classes. I still do `getattr(op, "_generated", False)` just in case, but I am not fond of missing fields.

Reply:
Meh. I'll not sweat it, but I disagree on adding this kind of field "for completeness". It adds noise and creates yet another place where the code can go stale.

Reply:
Tbh, there is no actual base class for all ops, at least as far as I understand Python 🙃 
And the hand written ones don't inherit from the one in op_factory, there is a TODO to fix it, as it would eliminate even more common code, but I decided not to spend another week on reworking that and introduce the actual hints first.